Sources/Parser.swift

```diff
@@ -47,7 +48,11 @@ public class TokenParser {
           return nodes
         }
 
-        if let tag = token.components().first {
+        let components = token.components()
+        if var tag = components.first {
+          if tag.hasSuffix(":") && components.count >= 2 {
```

Comment:
If we're keeping the label syntax (see question above), we should add a comment here what's happening and why.